example/app.js

```diff
@@ -4,18 +4,21 @@ class PostModel extends BaseModel {
   constructor (root) {
     super(root)
     console.log('ROOT.form_data', root.form_data)
-    this
-      .addContainer(
-        'user', {
+    this.addContainers({
```

Comment:
So, what do you think about this, @kaskar2008?

Reply:
It's just I'm comparing to the ES6 version without source in the container, so it looks nicer :)
```js
{
  'user': {
    'field1': 'type1',
    'field2': 'type2',
  }
}
```

But ok, this one is fine